spotbugs-tests/src/test/java/edu/umd/cs/findbugs/detect/Issue374Test.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Contributions to SpotBugs
+ * Copyright (C) 2020, kengo
```

Comment:
Use your name, please :)

Reply:
I just copied from another place and didn't notice that XD. 
Can you correct this copyright owner in your repository @cwbriones? You can use "ritchan". Thanks!